<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1580</width>
    <height>909</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="4">
         <widget class="QPushButton" name="StopButton">
          <property name="text">
           <string>Stop</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Task1_signal viewer/stop.png</normaloff>../../../Task1_signal viewer/stop.png</iconset>
          </property>
         </widget>
        </item>
        <item row="7" column="8">
         <widget class="QLabel" name="label_4">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Max</string>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QSlider" name="verticalSlider_11">
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="4" column="5" colspan="4">
         <widget class="PlotWidget" name="graphicsView_outputsignal"/>
        </item>
        <item row="4" column="0" colspan="5">
         <widget class="PlotWidget" name="graphicsView_inputsignal"/>
        </item>
        <item row="6" column="6">
         <widget class="QPushButton" name="ZoomoutButton">
          <property name="text">
           <string>Zoom out</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Task1_signal viewer/zoom out.png</normaloff>../../../Task1_signal viewer/zoom out.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="BrowseButton">
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Browse</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../Task1_signal viewer/browse.png</normaloff>../../Task1_signal viewer/browse.png</iconset>
          </property>
         </widget>
        </item>
        <item row="7" column="0" colspan="2">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Min</string>
          </property>
         </widget>
        </item>
        <item row="5" column="0" colspan="9">
         <widget class="QSlider" name="horizontalSlider">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="6">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Output Signal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="5">
         <widget class="QPushButton" name="ZoominButton">
          <property name="text">
           <string>Zoom in</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Task1_signal viewer/zoom in.png</normaloff>../../../Task1_signal viewer/zoom in.png</iconset>
          </property>
         </widget>
        </item>
        <item row="1" column="4" colspan="2">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600;&quot;&gt;Sound Equlizer&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="6" column="3">
         <widget class="QPushButton" name="StartButton">
          <property name="text">
           <string>Start</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>../../../Task1_signal viewer/start.png</normaloff>../../../Task1_signal viewer/start.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QPushButton" name="insert">
          <property name="text">
           <string>+</string>
          </property>
         </widget>
        </item>
        <item row="8" column="8">
         <widget class="QSlider" name="verticalSlider_12">
          <property name="minimum">
           <number>50</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
         </widget>
        </item>
        <item row="3" column="2">
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <pointsize>14</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;right&quot;&gt;Input Signal&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="8" column="1">
         <widget class="QComboBox" name="colorpalette">
          <property name="font">
           <font>
            <pointsize>9</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <item>
           <property name="text">
            <string>inferno</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>viridis</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>plasma</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>cool</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>PuBu</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>magma</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>YlGnBu</string>
           </property>
          </item>
         </widget>
        </item>
        <item row="7" column="2" rowspan="2" colspan="5">
         <widget class="mplwidget" name="graphicsView_spectrogram"/>
        </item>
        <item row="9" column="1" colspan="7">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QSlider" name="verticalSlider_1">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_2">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_3">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_4">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_5">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_6">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_7">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_8">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_9">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="verticalSlider_10">
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>1</number>
            </property>
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1580</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>mplwidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">mplwidget.h</header>
  </customwidget>
  <customwidget>
   <class>PlotWidget</class>
   <extends>QGraphicsView</extends>
   <header location="global">pyqtgraph</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
